src/model/immutable/ContentBlockNode.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ContentBlockNode
+ * @format
+ * @flow
+ *
+ * This is unstable and not part of the public API and should not be used by
+ * production systems. This file may be update/removed without notice.
+ */
+
+'use strict';
+
+const CharacterMetadata = require('CharacterMetadata');
+const findRangesImmutable = require('findRangesImmutable');
+
+import type {BlockNode, BlockNodeConfig, BlockNodeKey} from 'BlockNode';
+import type {DraftBlockType} from 'DraftBlockType';
+import type {DraftInlineStyle} from 'DraftInlineStyle';
+
+import {List, Map, OrderedSet, Record, Repeat} from 'immutable';
+
+type ContentBlockNodeConfig = BlockNodeConfig & {
+  nodes?: List<BlockNodeKey>,
```

Comment:
`nodes` are the children, right? What do you think of `children` or `childNodes`?